python/cudf/cudf/core/udf/_ops.py

```diff
@@ -53,3 +61,14 @@
     operator.gt,
     operator.ge,
 ]
+
+
+# currently only numeric types are supported.
+SUPPORTED_TYPES = NUMERIC_TYPES | BOOL_TYPES | DATETIME_TYPES | TIMEDELTA_TYPES
+
+
+def _is_jit_supported_type(dtype):
```

Comment:
Until this function seems like having a broader use case it might make more sense to just define it in pipeline.py. This extra indirection doesn't buy us anything.

Reply:
Moved